### PR TITLE
Random product

### PR DESCRIPTION
**Pull request by Djekkson12**
Randomizer + cucumber + some xpaths

### CODE REVIEW

File: src/main/java/pageObjects/HomePage.java [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+
+@Getter
+@Setter
+public class HomePage extends AbstractPage {
+    public HomePage() {
+        super();
+    }
+
+    @FindBy(xpath = "//button[@class='button button_serv js-delivery-menu']")
+    private static WebElement MenuButton;
```
Comment (AlexandrBotnari) [inline src/main/java/pageObjects/HomePage.java]: Please  remove static and rename elements to lowercase

File: src/main/java/pageObjects/MenuPage.java [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+
+@Getter
+@Setter
+
+public class MenuPage extends AbstractPage {
+    public MenuPage() {
+        super();
+    }
+
+    @FindBy(xpath = "//div[@class='modal-menu modal-menu_open']//a[*]")
```
Comment (AlexandrBotnari) [inline src/main/java/pageObjects/MenuPage.java]: redo  from //a[*] =//a

File: src/main/java/pageObjects/MenuPage.java [hunk 3]
```diff
@@ -1,8 +1,44 @@
...
+    private static List<WebElement> MenuList;
+
+
+    @FindBy(xpath = "/html[1]/body[1]/div[4]/div[2]/div[1]/div[*]/div[4]")
```
Comment (AlexandrBotnari) [inline src/main/java/pageObjects/MenuPage.java]: Write another xpath like //div[@bloc='']/bloc  an so on

File: src/test/java/hooks/Hooks.java [hunk 4]
```diff
@@ -15,12 +15,13 @@ public void startDriver() throws Exception {
...
         Driver.getDriver().manage().window().maximize();
+        Driver.getDriver().get("https://www.andys.md/en");
     }
 
 
-    @After
-    public void after() throws Exception {
-        Driver.getDriver().quit();
-    }
+//    @After
```
Comment (AlexandrBotnari) [inline src/test/java/hooks/Hooks.java]: uncomment hook

File: src/test/java/runners/RunCucumberTest.java [hunk 5]
```diff
@@ -8,10 +8,10 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(
-        features = "classpath:features/ChangeLanguge.features",
+        features = "classpath:features",
         plugin = {"pretty", "html:target/cucumber-html-report", "json:target/cucumber.json},"},
         glue = {"classpath:steps","classpath:hooks"},
-        tags = "@cl",
+        tags = "@aa",
```
Comment (acaramalac) [inline src/test/java/runners/RunCucumberTest.java]: This tag doesn't say anything

File: src/test/java/steps/RandomProductSelect_Steps.java [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    @And("^Selects random category from \"([^\"]*)\" list$")
+    public void SelectRandomCategoryFromMenu() {
+        GetRandomCategory();
+    }
+
+    @Then("^User adds random product from category to cart$")
+    public void SelectRandomProduct() throws InterruptedException {
+        GetRandomProduct();
+        Thread.sleep(2000);
+        HomePage.CartCounterAssertCheck();
```
Comment (acaramalac) [inline src/test/java/steps/RandomProductSelect_Steps.java]: Try to avoid usage of Thread.sleep here

**Pull request closed without merging**